packages/plexus/package.json

```diff
@@ -3,7 +3,7 @@
   "license": "Apache-2.0",
   "version": "0.2.0",
   "description": "Directed Graph React component",
-  "main": "lib/index.js",
+  "main": "src/index.tsx",
```

Comment:
hold on, I don't think this is the right fix. When I talked about overrides I was referring to jaeger-ui package, similar to the other import fixes you did. But if you make this change in plexus then it breaks it as a standalone library, because once it's published (e.g. to npm) only the /lib folder is included in the bundle, not the sources (as far as I understand). Even if sources are included, it would not be usable from pure JS projects.

Reply:
For example, Bard things there are multiple ways to do that: https://g.co/bard/share/12a255963a6f

Reply:
Yes, you are right the src won't be included, even if it is included it won't be compatible because it is written in TypeScript. But, this was the only possible override which pointed all the imports to src and not lib. Changing it back to lib, I will have to individually modify all the import statements

Reply:
no, don't change imports, try to look into override options

Reply:
Got it will look into this